src/main/scala/uk/gov/dhsc/htbhf/RunGatlingTests.scala.tmp

```diff
@@ -19,7 +19,7 @@ object RunGatlingTests {
     try {
       val result = Gatling.fromMap(config)
       // write the result to system.out so we can identify success or failure from the logs
-      System.out.println("Finished running gatling tests - result=" + result)
+      System.out.println("Finished running gatling tests with feature toggles: " + sys.env.get("FEATURE_TOGGLES") + " - result=" + result)
```

Comment:
Can we log the feature toggles on a separate line? There could eventually be quite a few of them. Also, the CD scripts look for this _exact_ string: "Finished running gatling tests - result="

Reply:
Actually, while we're there, we should probably add a comment to that effect

Reply:
Added log to it's own line and added comment about the line not too edit.